typing.js

```diff
@@ -0,0 +1,47 @@
+import { getWordsFromApi } from "./getWordsFromApi.js";
+
+const textarea = document.querySelector(".main__textarea");
+const inputfield = document.querySelector(".main__inputfield");
+const errorMessage = document.querySelector(".input__error");
+const newTextButton = document.querySelector("#newTextButton");
+
+var text;
+
+const MAXWORDS = 30;
+var textLength = 10;
+
+async function getRandomText() {
+  text = await getWordsFromApi(textLength);
+  textarea.innerHTML = text;
+  errorMessage.innerHTML = "";
+}
+
+function checkIfEnter(event) {
+  if (event.key === "Enter") {
+    checkTextLength();
+    inputfield.blur();
+  }
+  else {
+    return;
+  }
```

Comment:
```suggestion

  return;
```

or better yet, use an early return

```suggestion
function checkIfEnter(event) {
  if (event.key !== "Enter") {
    return;
  }
  
  checkTextLength();
  inputfield.blur();
}
```